src/components/SummaryList.tsx

```diff
@@ -51,23 +55,49 @@ export const SummaryList: React.FunctionComponent<SummaryListProps> = (
 
   const extraAttributes = Attributes.ariaAndData(props);
 
+  const anyRowHasActions = rows.some(
+    row => row.actions && row.actions.length > 0
```

Comment:
Tip for next time:

```suggestion
    row => row.actions?.length
```

That would do the job.